### ISSUE TITLE
SetRecordingFolder: what this method does?

### ISSUE THREAD

**Issue by aaarghhh** [Palakis/obs-websocket#233]
##### Issue type
Other?

##### Description
Invoking this method it create a an absolute directory in the filesystem and set the mount point of the  setting' fileexplorer in that directory, so i don't understand what this method is intended to be. 
You have to update the directory using obs'setting otherwise when you start recording the file will be created in the previous directory. 

##### Technical information
- **WIN 10** :
- **21.1.2** :

**Comment by Palakis** [comment]
@aaarghhh I tested this again and I don't see major issues with it

Here's what I did, both for Simple and Advanced output modes:

1. Create two folders: "OBS Test 1" and "OBS Test 2" (simple names for simplicity, the next steps will assume these are absolute paths)
2. Through the settings dialog, set the recording path to "OBS Test 1"
3. Start recording
4. Notice the recorded file in "OBS Test 1"
5. Stop recording
6. Without touching the settings dialog, call SetRecordingFolder with "OBS Test 2"
7. Start recording
8. Notice the recorded file in "OBS Test 2"
9. Stop recording

Hope this helps!

**Comment by Palakis** [comment]
@aaarghhh Gonna make some changes to the documentation of SetRecordingFolder as well as simplify a bit how the recording path is set in OBS settings (most notably: set the path for Simple and Advanced modes at the same time)

**Comment by aaarghhh** [comment]
Thank you for your feedback. I want to tell that the issue is related to a condition that I found a long time ago so I don't rememeber when it happened.  The project I built was a cross platform app which can record in win/linux environment. If I didn't forgot something the bug happend only in one environment. I have to check which one was affected. 

I think that the issue could be related to the OBS version also, so it will be quite hard to reproduce. Sorry for for the few details I provided .
